go.mod

```diff
@@ -6,33 +6,33 @@ require (
 	cloud.google.com/go/storage v1.14.0
 	github.com/Masterminds/semver v1.5.0
 	github.com/Masterminds/sprig v2.22.0+incompatible
-	github.com/ahmetb/gen-crd-api-reference-docs v0.2.0
+	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
 	github.com/coreos/go-systemd/v22 v22.3.2
-	github.com/gardener/etcd-druid v0.15.0
-	github.com/gardener/gardener v1.62.0
+	github.com/gardener/etcd-druid v0.15.3
+	github.com/gardener/gardener v1.63.1-0.20230131115839-8d53342ddcf0
 	github.com/gardener/gardener-extension-networking-calico v1.27.1
 	github.com/gardener/gardener-extension-networking-cilium v1.18.0
 	github.com/gardener/machine-controller-manager v0.45.0
 	github.com/go-logr/logr v1.2.3
 	github.com/golang/mock v1.6.0
-	github.com/onsi/ginkgo/v2 v2.1.6
-	github.com/onsi/gomega v1.21.1
-	github.com/spf13/cobra v1.4.0
+	github.com/onsi/ginkgo/v2 v2.6.1
+	github.com/onsi/gomega v1.24.2
+	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/oauth2 v0.0.0-20220411215720-9780585627b5
-	golang.org/x/tools v0.1.12
+	golang.org/x/tools v0.4.0
 	google.golang.org/api v0.74.0
-	k8s.io/api v0.25.2
-	k8s.io/apiextensions-apiserver v0.25.0
-	k8s.io/apimachinery v0.25.2
+	k8s.io/api v0.26.1
+	k8s.io/apiextensions-apiserver v0.26.1
+	k8s.io/apimachinery v0.26.1
 	k8s.io/autoscaler/vertical-pod-autoscaler v0.12.0
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/code-generator v0.25.0
-	k8s.io/component-base v0.25.0
-	k8s.io/kubelet v0.25.0
-	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
-	sigs.k8s.io/controller-runtime v0.13.0
-	sigs.k8s.io/controller-tools v0.10.0
+	k8s.io/code-generator v0.26.1
+	k8s.io/component-base v0.26.1
+	k8s.io/kubelet v0.26.1
+	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
+	sigs.k8s.io/controller-runtime v0.14.1
```

Comment:
Is it possible to go to v0.14.4 right away?

Reply:
When we vendor `gardener@1.64.0`, it'll be `v0.14.4`, right?